tests/generation/test_hub.py

```diff
@@ -34,17 +34,44 @@ def test_model_from_hub():
 
 @is_inferentia_test
 @requires_neuronx
-def test_push_to_hub(neuron_model_path, neuron_push_id):
-    model = NeuronModelForCausalLM.from_pretrained(neuron_model_path)
-    model.push_to_hub(neuron_model_path, neuron_push_id, use_auth_token=TOKEN, endpoint=ENDPOINT_STAGING)
+def test_push_to_hub(neuron_decoder_path, neuron_push_decoder_id):
```

Comment:
You can make this test a staging test.

Reply:
Isn't it already a staging test pushing to the stage hub with `ENDPOINT_STAGING`?

Reply:
Oh yes it might, my bad, I do it in a different way.